src/main/java/com/hedera/hashgraph/sdk/QueryBuilder.java

```diff
@@ -109,11 +109,25 @@ public T setPaymentDefault(long paymentAmount) {
         return (T) this;
     }
 
-    public final long requestCost() throws HederaException, HederaNetworkException {
+    public final long getCost() throws HederaException, HederaNetworkException {
         return new CostQuery().execute();
     }
 
-    public final void requestCostAsync(Consumer<Long> withCost, Consumer<HederaThrowable> onError) {
+    public final void getCostAsync(Consumer<Long> withCost, Consumer<HederaThrowable> onError) {
+        new CostQuery().executeAsync(withCost, onError);
+    }
+
+    /**
+     * @deprecated renamed to {@link #getCost()}
+     */
+    public final long queryCost() throws HederaException, HederaNetworkException {
+        return new CostQuery().execute();
+    }
+
+    /**
+     * @deprecated renamed to {@link #getCostAsync(Consumer, Consumer)}
+     */
+    public final void queryCostAsync(Consumer<Long> withCost, Consumer<HederaThrowable> onError) {
```

Comment:
Why are there 2 new methods added that are deprecated?

Reply:
Initial refactor that I forgot to revert. Fixed.